proof_of_stake/src/lib.rs

```diff
@@ -327,7 +322,7 @@ where
             current_epoch,
             epoch,
         )?;
-        store_total_consensus_stake(storage, epoch)?;
+        // compute_and_store_total_consensus_stake(storage, epoch)?;
```

Comment:
Why is this now commented?

Reply:
It was a redundant operation. We already call this function in `init_chain` and in `finalize_block` upon a new epoch, both times before calling `copy_validator_sets`. It is only necessary to call the function at the beginning of a new epoch and compute the total consensus stake for that current epoch only.